Src/PChecker/CheckerCore/CheckerCore.csproj

```diff
@@ -6,5 +6,6 @@
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
         <OutputPath>$(PSdkFolder)\Binaries</OutputPath>
         <TargetFramework>net6.0</TargetFramework>
+        <Version>2.0.4</Version>
```

Comment:
Why are we hard-coding it here?

Reply:
Corrected in f4e0178a4f0df7847656d5262104f2154ed71df8